lib/stream.js

```diff
@@ -86,14 +106,24 @@ const applyEncoding = async (stream, maxBuffer, encoding) => {
 	return buffer.toString(encoding);
 };
 
+// Handle any errors thrown by the iterable passed to the `stdin` option, if any.
+// We do not consume nor wait on that stream though, since it could potentially be infinite (like `process.stdin` in an interactive TTY).
+const throwOnStreamsError = streams => streams.filter(Boolean).map(stream => throwOnStreamError(stream));
+
+const throwOnStreamError = async stream => {
+	const [error] = await once(stream, 'error');
+	throw error;
+};
```

Comment:
This could technically be just:

```js
const throwOnStreamError = stream => once(stream, '');
```

Since `once()` throws on `error` events, and stream events are never an empty string. But this would make the code harder to understand.